index.html

```diff
@@ -80,17 +80,17 @@
 
           <div class="card row col-xs-5">
             <div class="info">
-              <h3>REVO</h3>
-              <p id="description">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+              <h4>REVO <small> Research of Electric Vehicles at Olin </small></h4>
+              <p id="description">A team that researches and builds electric vehicles at Olin. </p>
             </div>
 
             <div class="contact">
               
               <ul class= "links list-unstyled">
-                <li> <a id="website"><i class="fa fa-link"></i></a> </li>
-                <li> <a id="email"><i class="fa fa-envelope-o"></i></a> </li>
-                <li> <a id="facebook"><i class="fa fa-facebook"></i></a> </li>
-                <li> <a id="twitter"><i class="fa fa-twitter"></i></a> </li>
+                <li> <a href="http://www.olinrevo.org/"><i class="fa fa-link"></i></a> </li>
```

Comment:
These probably shouldn't have spaces between the `<li>` and `<a>` tags. Were you using these to get more padding? We can get the padding with CSS.